internal/cmd/connect/command_custom_plugin_describe.go

```diff
@@ -0,0 +1,51 @@
+package connect
+
+import (
+	pcmd "github.com/confluentinc/cli/internal/pkg/cmd"
+	"github.com/confluentinc/cli/internal/pkg/examples"
+	"github.com/confluentinc/cli/internal/pkg/output"
+	"github.com/spf13/cobra"
+)
+
+func (c *customPluginCommand) newDescribeCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:         "describe <id>",
+		Short:       "Describe a custom connector plugin.",
+		Args:        cobra.ExactArgs(1),
+		RunE:        c.describe,
+		Annotations: map[string]string{pcmd.RunRequirement: pcmd.RequireNonAPIKeyCloudLogin},
```

Comment:
This shouldn't be needed if the parent has the same annotation (I think that's what you should be doing...)